test/common/upstream/upstream_impl_test.cc

```diff
@@ -2235,6 +2235,45 @@ TEST_F(ClusterInfoImplTest, OneofExtensionProtocolOptionsForUnknownFilter) {
                             "Only one of typed_extension_protocol_options or "
                             "extension_protocol_options can be specified");
 }
+
+TEST_F(ClusterInfoImplTest, TestTrackRequestResponseSizes) {
+  const std::string yaml_disabled = R"EOF(
+    name: name
+    connect_timeout: 0.25s
+    type: STRICT_DNS
+    lb_policy: ROUND_ROBIN
+  )EOF";
+
+  auto cluster = makeCluster(yaml_disabled);
+  // By default, histograms tracking request/response sizes are not published.
+  EXPECT_FALSE(cluster->info()->requestResponseSizeStats().has_value());
```

Comment:
do we also need to check that tracking of timeout budgets is not enabled?

Reply:
Nice Catch. Thanks.

Reply:
on second thought, we don't need to check that tracking of timeout budgets is not enabled here as it is already checked in `TestTrackTimeoutBudgets`.